MODULE.bazel

```diff
@@ -8,15 +8,11 @@ module(
 
 bazel_dep(name = "aspect_bazel_lib", version = "1.27.2")
 bazel_dep(name = "bazel_skylib", version = "1.4.1")
-bazel_dep(name = "rules_nodejs", version = "5.8.0")
+bazel_dep(name = "rules_nodejs", version = "5.8.2")
```

Comment:
Including this dep is enough to register the node toolchain at the default version.

Reply:
yeah looks much better